### ISSUE TITLE
Find dialog transparency bug

### ISSUE THREAD

**Issue by justanotheranonymoususer** [notepad-plus-plus/notepad-plus-plus#3439]
### Description of the Issue
The find dialog becomes transparent when it doesn't need to.

### Steps to Reproduce the Issue
1. Open Find dialog with Ctrl+F.
2. Press Tab and hold it to navigate between controls
3. Note that eventually the "Transparency" -> "Always" radio button gets focus, and the dialog becomes transparent.

**Comment by dinkumoil** [comment]
I can confirm this behaviour under certain circumstances.

Steps to reproduce:

1. Start a new instance of Notepad++.
2. Open "Find" dialog with Ctrl+F.
3. Confirm that in the "Transparency" group box the radio button "On losing focus" is selected. If this is not the case, select it, close the "Find" dialog, restart Notepad++ and reopen the "Find" dialog.
4. Press tab key to navigate between controls.
5. One can notice that it is possible to set the focus rectangle to the "Always" radio button in the "Transparancy" group box. Once this has happend the "Find" dialog gets transparent. In contrast to normal transparency mode of this dialog the title bar does not change its color to white, it turns into transparent blue instead (because the dialog still has the focus).
6. When the "Find" dialog is closed while the "Always" radio button is **not selected but focused** (like on the image below) and Notepad++ is closed too, **after next restart** of Notepad++ the transparency preference is set to "Always".

![grafik](https://user-images.githubusercontent.com/24699812/27515007-13cb920c-599a-11e7-9801-a13267c319bf.png)

Because the "Transparency" group box contains a group of radio buttons it should not be possible to set the focus rectangle to the **unselected** "Always" radio button. Normally it is only possible to set the focus rectangle to the **selected** button of such a group.

When I tried to reproduce the bug I noticed that it doesn't occur always. But immediately after starting Notepad++ I was able to reproduce it every time.
